<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagRegisterWidget</class>
 <widget class="QWidget" name="TagRegisterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>タグ登録</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEditTag">
     <property name="placeholderText">
      <string>タグ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSourceTag">
     <property name="placeholderText">
      <string>元タグ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxFormat">
     <property name="placeholderText">
      <string>フォーマット</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxType">
     <property name="placeholderText">
      <string>タイプ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxUseCount">
     <property name="prefix">
      <string>使用回数: </string>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxLanguage">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>言語</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditTranslation">
     <property name="placeholderText">
      <string>翻訳</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonRegister">
     <property name="text">
      <string>登録</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditOutput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
